transit_test.go

```diff
@@ -238,3 +238,73 @@ func (s *TransitTestSuite) TestCreateKeyThatDoesAlreadyExist() {
 	err = s.client.Create("testCeateKeyThatDoesAlreadyExist", &TransitCreateOptions{})
 	require.NoError(s.T(), err)
 }
+
+func (s *TransitTestSuite) TestImplicitSignCreate() {
+	_, err := s.client.Sign("test404", &TransitSignOptions{
```

Comment:
Try to use a different key name like `testImplicitSignCreate` to ensure a new key is generated